setup.py

```diff
@@ -21,18 +24,25 @@
 with open('README.rst') as f:
     readme = f.read()
 
+PY3 = (sys.version_info[0] >= 3)
+
+install_requires = [
+    "pyOpenSSL >= 0.13",
+    "requests >= 2.1.0",
+    "service_identity >= 14.0.0",
+    "six"
+]
+
+if PY3:
```

Comment:
Choose Your Own Adventure Style:

You are in a dark and windy maze, on the right is a door labeled "I care about versions of setuptools older than 0.7 and versions of pip older than 6.0" on the left is a door that says "Only the latest of versions are good enough for me".

If you take the left door, delete the ``if PY3`` conditional and use the syntax @hynek posted and find the escape and quit reading.

If you take the right door, you then find yourself in a circular room, a door on the other side is labeled "I don't want to duplicate my dependency information and two wheel files is fine", in the middle of the room is a ladder leading into the darkness labeled "I want a single wheel file and I'm OK having to list all of my dependencies twice".

If you flee the darkness and exit through the door delete ``universal=1`` from the ``setup.cfg`` and escape.
If you descend the ladder, then buy volume 2 where you'll get an ancient tome that you can cargo cult into the proper locations.

tl;dr

Important Question: Do you care if you have 1 wheel file or two for each version, and do you want to support versions of pip < 6.0 and versions of setuptools < 0.7.